subsys/bootloader/Kconfig

```diff
@@ -55,13 +55,19 @@ config SB_NUM_VER_COUNTER_SLOTS
 
 endif # SECURE_BOOT
 
+#hidden option for mark secure-boot as nRF54L15 style
+config SB_NRF54L15_STYLE_IMPL
+	bool
+	default y if SOC_NRF54L15_CPUAPP || SOC_NRF54L05_CPUAPP || SOC_NRF54L10_CPUAPP
```

Comment:
do not add a new symbol, use `SOC_SERIES_NRF54LX`

Reply:
SOC_SERIES_NRF54LX covers also LM devices which are not planed to be supported in the same manner as nRF54l15

Reply:
see also https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/nrf_security/src/drivers/cracen/Kconfig#L11